src/common/colorspaces.c

```diff
@@ -1176,6 +1189,11 @@ dt_colorspaces_t *dt_colorspaces_init()
                                                                dt_colorspaces_create_adobergb_profile(),
                                                                _("Adobe RGB (compatible)"),
                                                                ++in_pos, ++out_pos, ++display_pos, ++category_pos, ++work_pos));
+                                                               
+  res->profiles = g_list_append(res->profiles, _create_profile(DT_COLORSPACE_REC709,
+                                                               dt_colorspaces_create_gamma_rec709_rgb_profile(),
+                                                               _("gamma Rec709 RGB"),
+                                                               -1, ++out_pos, ++display_pos, -1, -1));
```

Comment:
Are you sure you want this on the display and not in the work category?

Reply:
I'm not sure... :-)
Some 3D luts are built to be applied on this profile. There may be some other transformations to be  done on that profile.

Reply:
OK, but that has nothing to do with the display group, and since the profile is there maybe someone can use it as a work profile.